Documentation/network/kubernetes/ipam-multi-pool.rst

```diff
@@ -0,0 +1,194 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    http://docs.cilium.io
+
+.. _gsg_ipam_crd_multi_pool:
+
+*******************************************
+CRD-Backed by Cilium Multi-Pool IPAM (Beta)
+*******************************************
+
+This is a quick tutorial walking through how to enable multi-pool IPAM backed by the
+``CiliumPodIPPool`` CRD. The purpose of this tutorial is to show how components are configured and
+resources interact with each other to enable users to automate or extend on their own.
+
+For more details, see the section :ref:`ipam_crd_multi_pool`
+
+Enable Multi-pool IPAM mode
+===========================
+
+#. Setup Cilium for Kubernetes using helm with the options:
+
+   * ``--set ipam.mode=multi-pool``
+   * ``--set autoDirectNodeRoutes=true``
+   * ``--set ipv4NativeRoutingCIDR=10.0.0.0/8``
+   * ``--set endpointRoutes.enabled=true``
+   * ``--set-string extraConfig.enable-local-node-route=false``
+   * ``--set kubeProxyReplacement=strict``
+   * ``--set bpf.masquerade=true``
```

Comment:
I think `--set tunnel=disabled` is missing here.

Reply:
Nice catch, added.